src/main/docs/guide/quickStart.adoc

```diff
@@ -1,10 +1,10 @@
 Add the following dependency to your Micronaut application:
 
-dependency:io.micronaut.reactor:micronaut-reactor[scope="compile", version="{version}"]
+dependency:micronaut-reactor[scope="compile", groupId="io.micronaut.reactor", version="{version}"]
```

Comment:
this generates 

```
implementation("io.micronaut.reactor:micronaut-reactor:2.2.1-SNAPSHOT")
```

I think what we should have is: 

```
implementation("io.micronaut.reactor:micronaut-reactor")
```

which we could achieve with: 

```
dependency:micronaut-reactor[groupId="io.micronaut.reactor"]
```